src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -210,19 +215,19 @@ int ACA_Dataplane_OVS::update_port_state_workitem(const PortState current_PortSt
       port_cidr = virtual_ip_address + "/" + found_prefix_len;
 
       if (overall_rc == EXIT_SUCCESS) {
-        ACA_LOG_DEBUG("Port Operation: %s: project_id:%s, vpc_id:%s, network_type:%d, virtual_ip_address:%s, "
-                      "virtual_mac_address:%s, port_name: %s, port_cidr: %s, tunnel_id: %d\n",
+        ACA_LOG_DEBUG("Port Operation: %s: port_id: %s, project_id:%s, vpc_id:%s, network_type:%d, "
+                      "virtual_ip_address:%s, virtual_mac_address:%s, port_name: %s, port_cidr: %s, tunnel_id: %d\n",
```

Comment:
do you mean subnet_cidr, instead of port_cidr?

Reply:
it is the port CIDR notation, e.g. 10.0.0.5/24.